### PR TITLE
Major v11

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by notlee** [comment]
released as v11.0.0-beta.1

**Comment by JakeChampion** [comment]
We need to release a new version of [create-origami-component](https://github.com/Financial-Times/create-origami-component/issues/220) before releasing this new version of origami build tools.

File: lib/tasks/verify-package-json.js [hunk 1]
```diff
@@ -95,6 +95,10 @@ async function packageJson(config) {
 	if (exists) {
 		const file = await readFile(packageJsonPath, 'utf8');
 		const packageJson = JSON.parse(file);
+
+		if (packageJson.type !== 'module') {
```
Comment (notlee) [inline lib/tasks/verify-package-json.js:99]: Are we not dropping `module` for `component` in v2? Currently an alias.
https://origami.ft.com/spec/v1/manifest/#origamitype

Comment (notlee) [reply lib/tasks/verify-package-json.js:99]: https://github.com/Financial-Times/origami-website/pull/273/files#r617396987

Comment (JakeChampion) [reply lib/tasks/verify-package-json.js:99]: > Are we not dropping `module` for `component` in v2? Currently an alias.
> [origami.ft.com/spec/v1/manifest/#origamitype](https://origami.ft.com/spec/v1/manifest/#origamitype)

This is the package.json manifest, the value of the `type` field specifies to NodeJS and build tools whether the package is written using `require` or `import/export`.

Comment (notlee) [reply lib/tasks/verify-package-json.js:99]: Ahhh of course, good, well at least it got me thinking about `origamiType` :)

**Review by JakeChampion — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v11.0.0 :tada:

**Pull request merged**